tests/manage/pv_services/test_pvc_rwx_writeable_after_pod_deletions.py

```diff
@@ -0,0 +1,166 @@
+import logging
+import pytest
+
+from concurrent.futures import ThreadPoolExecutor
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources import pod
+from tests import helpers
+
+logger = logging.getLogger(__name__)
+
+
+def verify_pod_node(pod_obj, node_name):
```

Comment:
I think you can use ocs_ci.ocs.resources.pod.get_pod_node and check whether the node name returned by the function is the particular node or not from the test itself

Reply:
It appears ocs_ci.ocs.resources.pod.get_pod_node is returning node object, won't that be an overhead and cause code duplication everytime just for checking the nodename of a pod?